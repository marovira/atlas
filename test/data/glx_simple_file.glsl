#version 450 core

layout(location = 0) in vec4 pos;

layout(std140, binding = 0) uniform Matrices
{
    mat4 projection;
    mat4 view;
}

uniform mat4 model;

void main()
{
    gl_Position = projection * view * model * vec4(pos.xyz, 1.0);
}
