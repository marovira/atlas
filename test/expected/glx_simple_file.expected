#version 450 core
#line 2 h

#line 3 h
layout(location = 0) in vec4 pos;
#line 4 h

#line 5 h
layout(std140, binding = 0) uniform Matrices
#line 6 h
{
#line 7 h
    mat4 projection;
#line 8 h
    mat4 view;
#line 9 h
};
#line 10 h

#line 11 h
uniform mat4 model;
#line 12 h

#line 13 h
void main()
#line 14 h
{
#line 15 h
    gl_Position = projection * view * model * vec4(pos.xyz, 1.0);
#line 16 h
}
